### ISSUE TITLE
Joining #japanese:matrix.org via URL in Vector develop crashes the client

### ISSUE THREAD

**Issue by ara4n** [vector-im/vector-web#888]
https://vector.im/develop/#/room/#japanese:matrix.org

=> infinite spinner and

```
bundle.js:1 Restoring session for @matthew:matrix.org
2016-02-08 17:25:19.645 bundle.js:1 Guest: false
2016-02-08 17:25:19.792 bundle.js:107 SyncApi.sync
2016-02-08 17:25:22.018 sync.js:48 Got push rules
2016-02-08 17:25:22.019 client.js:3154 Got TURN URIs: turn:turn.matrix.org:3478?transport=udp,turn:turn.matrix.org:3478?transport=tcp,turns:turn.matrix.org:443?transport=tcp refresh in 86400 secs
2016-02-08 17:25:22.019 sync.js:48 Using existing filter ID 15
2016-02-08 17:25:22.872 Presence.js:83 Presence: online
2016-02-08 17:26:14.540 MatrixChat.js:537 MatrixClient sync state => PREPARED
2016-02-08 17:26:14.666 RoomView.js:166 Attempting to peek into room !TXthShfxHhyIaDmhvg:matrix.org
2016-02-08 17:26:14.872 InviteMemberList.js:46 Uncaught TypeError: Cannot read property 'hasMembershipState' of null(anonymous function) @ InviteMemberList.js:46
2016-02-08 17:26:16.840 index.js:218 GET https://matrix.org/_matrix/client/r0/rooms/!TXthShfxHhyIaDmhvg%3Amatrix.org… 403 (Forbidden)
2016-02-08 17:26:18.106 invariant.js:35 Uncaught Error: Minified exception occurred
2016-02-08 17:29:07.865 Presence.js:83 Presence: unavailable
2016-02-08 18:15:21.750 updater.js:41 Current=vector-27791c06ee27-react-a2e50eb496f1-js-977e33f1bd5c Latest=vector-27791c06ee27-react-4996164b46e8-js-977e33f1bd5c
2016-02-08 18:15:21.752 invariant.js:42 Uncaught Error: Invariant Violation: Dispatcher.unregister(...): `undefined` does not map to a registered callback.n @ invariant.js:42
```

**Comment by richvdh** [comment]
I think this has been fixed by matrix-org/matrix-react-sdk#150. Please reopen if you can still reproduce.

**Comment by richvdh** [comment]
Actually, let me rephrase that: please check if you can reproduce, and if you can't, assume it has been fixed and close it.

**Comment by ara4n** [comment]
can't repro - thanks :)
